app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookFragment.kt

```diff
@@ -87,9 +92,31 @@ class RoadBookFragment : Fragment(), MenuProvider {
 
         setLiveLocationEvent()
 
+        setShiftRepository()
+
         return view
     }
 
+    private fun setShiftRepository(){
+        if (userViewModel.loggedInUser.value != null){
+            if(rbViewModel.recordsListState.value != null) {
+                Log.i("RoadBookFragment", "Setting ShiftRepository")
+                val currentShift = Shift(
```

Comment:
This val is unused and it seems to be some debugging log in that function's body.

But that set function should not be used, the repository could be initilaized in the constructor arguments of the `RoadBookViewModel`, which is preferred and aligned to the `RoadBookRepository`.

(An other option is to instantiate it directly at his declaration line.)